.trunk/configs/.tflint.hcl

```diff
@@ -0,0 +1,98 @@
+plugin "terraform" {
+  enabled = true
+  preset  = "all"
+}
+
+config {
+  format = "compact"
+  call_module_type = "local"
+  force = false
+  disabled_by_default = false
+}
+
+# installing the aws ruleset from GitHub requires setting a GITHUB_TOKEN
+# environment variable. Without it, you'll get an error like this:
+#
+# Installing "aws" plugin...
+# Failed to install a plugin; Failed to fetch GitHub releases: GET https://api.github.com/repos/terraform-linters/tflint-ruleset-aws/releases/tags/v0.39.0: 401 Bad credentials []
+#
+# The solution is to provide a github PAT via a GITHUB_TOKEN env var,
+# export GITHUB_TOKEN=github_pat_120abc123def456ghi789jkl123mno456pqr789stu123vwx456yz789
+plugin "aws" {
+  enabled = true
+  version = "0.39.0"
+  source  = "github.com/terraform-linters/tflint-ruleset-aws"
+  deep_check = false
+}
+
+
+rule "terraform_comment_syntax" {
+    # Disallow `//` comments in favor of `#`
+    enabled = true
+}
+rule "terraform_deprecated_index" {
+    # Disallow legacy dot index syntax
+    enabled = true
+}
+rule "terraform_deprecated_interpolation" {
+    # Disallow deprecated (0.11-style) interpolation
+    # Enabled by default
```

Comment:
Let's remove those that are enabled by default. The smaller the config, the better.

Reply:
Good call. Removed the enabled by default blocks.